packages/block-editor/src/layouts/flex.js

```diff
@@ -36,6 +40,12 @@ const alignItemsMap = {
 	center: 'center',
 };
 
+const verticalAlignmentMap = {
+	top: 'flex-start',
+	middle: 'center',
```

Comment:
shouldn't this be like this?
```suggestion
	center: 'center',
```